src/components/InputWithTitle/InputWithTitle.tsx

```diff
@@ -0,0 +1,41 @@
+import { H1, Input, Stack, useDeskproAppTheme } from "@deskpro/app-sdk";
+import { UseFormRegisterReturn } from "react-hook-form";
+
+interface Props {
+  title: string;
+  error: boolean;
+  required?: boolean;
+  register: UseFormRegisterReturn;
+  type?: string;
+}
+
+export const InputWithTitle = ({
+  title,
+  error,
+  required,
+  register,
+  type,
+}: Props) => {
+  const { theme } = useDeskproAppTheme();
+  return (
+    <Stack vertical style={{ width: "100%", marginTop: "5px" }}>
+      <Stack>
+        <div style={{ color: theme.colors.grey80 }}>
+          <H1>{title}</H1>
+        </div>
+        {required && (
```

Comment:
Try to use the label with an asterisk use Label component, like this

```javascript
import { Label } from "@deskpro/deskpro-ui";

<Label label="this is label" required>
    <Input ... />
</Label>
```

Reply:
Label doesnt have the same properties as that one, and you cant apply a style on top of it.

Reply:
what styles do you need?

Reply:
A certain Size, color and weight, the same in Dropdown.

Reply:
We have a design system and it is implemented in deskpro-ui with all colors, size and weight. As it looks in desktop-ui, so it is correct. There they care about relevance with a design system - you and me don't need to think about appearance.

Reply:
Thing is, if you see a field that has title x and another field with title y, that will look extremely inconsistent.
I'm following the designs and am also making everything consistent.